jobs/webcompat-kb/webcompat_kb/bugzilla.py

```diff
@@ -0,0 +1,1366 @@
+        kb_bugs.update(
+            self.kb_bugs_from_platform_bugs(
+                platform_bugs, set(kb_bugs.keys()), set(site_reports.keys())
+            )
+        )
+
+        # Process KB bugs fields and get their dependant core/breakage bugs ids.
+        kb_data, kb_dep_ids = self.process_relations(kb_bugs, RELATION_CONFIG)
+        self.add_kb_entry_breakage(kb_data, kb_dep_ids, site_reports)
+
+        fetch_missing_result = self.fetch_missing_deps(all_bugs, kb_dep_ids)
+        if fetch_missing_result is None:
+            raise BugFetchError(
+                "Fetching missing dependencies from Bugzilla was not completed due to an error, aborting."
+            )
+
+        missing_bugs, core_missing = fetch_missing_result
+
+        platform_bugs.update(core_missing)
+        all_bugs.update(missing_bugs)
+
+        # Process core bugs and update KB data with missing links from core bugs.
+        if platform_bugs:
+            core_data, _ = self.process_relations(
+                platform_bugs, PLATFORM_RELATION_CONFIG
+            )
+            kb_data = self.add_links(kb_data, core_data)
+
+        # Build relations for BQ tables.
+        rels = self.build_relations(kb_data, RELATION_CONFIG)
+
+        kb_ids = list(kb_data.keys())
+
+        if self.include_history:
+            history_changes = self.fetch_bug_history(all_bugs)
+        else:
+            logging.info("Not updating bug history")
+            history_changes = []
+
+        etp_rels: Mapping[str, list[Mapping[str, Any]]] = {}
+        if etp_reports:
+            etp_reports_unified = self.unify_etp_dependencies(
+                etp_reports, etp_dependencies
+            )
+            etp_data, _ = self.process_relations(
+                etp_reports_unified, ETP_RELATION_CONFIG
+            )
+
+            etp_rels = self.build_relations(etp_data, ETP_RELATION_CONFIG)
+
+        if self.write:
+            self.update_history(history_changes)
+            self.update_bugs(all_bugs)
+            self.update_kb_ids(kb_ids)
+            self.update_relations(rels, RELATION_CONFIG)
+            self.update_relations(etp_rels, ETP_RELATION_CONFIG)
+
+            last_change_time_max = parse_datetime_str(
+                max(all_bugs.values(), key=lambda x: x["last_change_time"])[
+                    "last_change_time"
+                ]
+            )
+
+            self.record_import_run(
+                start_time,
+                self.history_fetch_completed,
+                len(all_bugs),
+                len(history_changes),
+                last_change_time_max,
+            )
+
+            self.update_metric_history()
+        else:
+            logging.info("Skipping writes")
+
+
+def add_arguments(parser: argparse.ArgumentParser) -> None:
+    group = parser.add_argument_group(
+        title="bugzilla", description="Bugzilla import arguments"
+    )
+    parser.add_argument(
+        "--bugzilla-api-key",
```

Comment:
Would need to add `dest="bugzilla_api_key"` here too, to match the underscore format in the [airflow config](https://github.com/mozilla/telemetry-airflow/blob/65f2bf0d8cb0cb3a42bc2a1fb0a50cd347e39e41/dags/webcompat_kb.py#L49).

Or we can just change all underscore arguments in airflow to the new format to avoid keeping the legacy ones.

Reply:
Oh actually never mind, I missed that it's reading it from the `BUGZILLA_API_KEY` env variable